source/grid/tria_accessor.cc

```diff
@@ -1555,6 +1566,18 @@ parent_index () const
 }
 
 
+
+template <int dim, int spacedim>
+unsigned int
+CellAccessor<dim, spacedim>::
+active_cell_index () const
+{
+  Assert (this->has_children()==false, TriaAccessorExceptions::ExcCellNotActive());
+  return this->tria->levels[this->present_level]->active_cell_indices[this->present_index];
```

Comment:
You mean whether the returned value is `!= numbers::invalid_unsigned_int`?

This would be an internal error: all active cells need to have a finite return value, and we assert that the cell is active (and by calling `this->has_children()` also that the cell is in fact used).